sentinel-adapter/sentinel-spring-webmvc-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/spring/webmvc/AbstractSentinelInterceptor.java

```diff
@@ -110,12 +108,13 @@ public void postHandle(HttpServletRequest request, HttpServletResponse response,
                            ModelAndView modelAndView) throws Exception {
     }
 
-    protected void setEntryInRequest(HttpServletRequest request, String name, Entry entry) {
+    protected void setEntryInRequest(HttpServletRequest request, String name, String resourceName) throws BlockException {
```

Comment:
Could you please add some comments regarding the related issue?

Reply:
I'll arrange it and add comments after work.

Reply:
Comments are added, please check it.